src/conq/navigation_lib/requirements.txt

```diff
@@ -0,0 +1,5 @@
+bosdyn-client==4.0.0
+openai==1.6.1
+python-dotenv==1.0.0
+numpy
+vtk
```

Comment:
Any reason why `navigation_lib` has its own set of requirements? Is this just so you don't have to worry about installing libraries that only Saket and Stanley need if you just want to prototype the navigation stuff?

If so, we can setup the TOML file to allow for pip install options that allow you to install only a subset of the library. That might be worth looking into (for me, not suggesting you do that for this PR)

Reply:
I used this originally when testing and I was calling everything from within the `navigation_lib` folder. I don't think this is explicitly needed now

Reply:
Got it. Your call on whether to leave it or delete it